pkg/exporter/process.go

```diff
@@ -305,6 +327,32 @@ func (ep *ExportingProcess) createAndSendMsg(set entities.Set) (int, error) {
 	return bytesSent, nil
 }
 
+// createAndSendJSONMsg takes in a set as input, creates the JSON record, and sends it out.
+func (ep *ExportingProcess) createAndSendJSONMsg(set entities.Set) (int, error) {
+	var bytesSent int
+	for _, record := range set.GetRecords() {
+		elements := make(map[string]interface{})
+		for _, element := range record.GetOrderedElementList() {
+			elements[element.Element.Name] = element.Value
+		}
+		message := make(map[string]interface{}, 2)
+		message["ipfix"] = elements
+		message["@timestamp"] = time.Now().Format(time.RFC3339)
+		writer := bytes.NewBuffer(make([]byte, 0, ep.jsonBufferLen))
+		encoder := json.NewEncoder(writer)
+		encoder.Encode(message)
```

Comment:
What happens if we message buffer in the encoder? Does it return error?

Reply:
you mean encode(writer)?

Reply:
I mean encoder.Encode function. Looked into it. I think we need to handle the error here.
https://github.com/square/go-jose/blob/v2.6.0/json/stream.go#L188